apps/site/assets/ts/leaflet/__tests__/MapTest.tsx

```diff
@@ -73,4 +74,18 @@ it("it renders using the default center position", () => {
       .prop("src")
   ).toBe(`${data.tile_server_url}/osm_tiles/16/19650/23738.png`);
 });
+
+it("it renders a marker with vehicle crowding info", () => {
+  const dataWithCrowding: MapData = {
+    ...data,
+    markers: [{ ...marker, vehicle_crowding: "some_crowding" }]
+  };
+  const div = document.createElement("div");
+  document.body.appendChild(div);
+  const wrapper = mount(<Map mapData={dataWithCrowding} />, {
+    attachTo: div
+  });
+
+  expect(wrapper.find(Marker)).toHaveLength(1);
```

Comment:
That's great, I didn't know you could just pass in a component instead of a selector!